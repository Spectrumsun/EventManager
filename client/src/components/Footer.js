import React from 'react';

/* const styles = {
  backgroundColor: '#35434a'
};
 */
const footer = () => (
  <div>
    <footer >
      <p>Event Managment &copy; 2017</p>
    </footer>
  </div>
);

export default footer;
