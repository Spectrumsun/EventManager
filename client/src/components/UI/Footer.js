import React from 'react';

/**
 * @description renders component to the DOM
 *
 * @function footer
 *
 * @returns {JSX} JSX component for the footer
 */
const footer = () => (
  <div className="center">
    <footer>
      <div><h3 className="center" >Event Manager</h3></div>
      <div style={{ display: 'inline-block', textAlign: 'left' }}>
        <div>
          <div className="float-left">
            <a className="navbar-brand" href="/"><img
              src="https://res.cloudinary.com/skybound/image/upload/v1522444986/eventmanager/static/event.png"
              width="30"
              height="30"
              alt=""
            />
            </a>
          </div>
          <div className="float-left">
            <ul >
              <p className="footerTitle">Social Media</p>
              <li className="footerColor">facebook</li>
              <li className="footerColor">twitter</li>
              <li className="footerColor">instargram</li>
            </ul>
          </div>
          <div className="float-left">
            <ul >
              <p className="footerTitle">Event manager</p>
              <li className="footerColor">Questions</li>
              <li className="footerColor">Jobs</li>
              <li className="footerColor">Help</li>
            </ul>
          </div>
          <div className="float-left">
            <ul >
              <p className="footerTitle">About</p>
              <li className="footerColor">desgin by Taiwo Sunday</li>
              <li className="footerColor">+2348038512455</li>
              <li
                className="footerColor"
              >https://github.com/Spectrumsun
              </li>
              <li
                className="footerColor"
              >taiwo.sunday@outlook.com
              </li>
            </ul>
          </div>
          <div className="float-left">
            <ul >
              <p className="footerTitle">Site</p>
              <li className="footerColor">Home</li>
              <li className="footerColor">Centers</li>
              <li className="footerColor">Events</li>
            </ul>
          </div>
        </div>
      </div>
      <p className="center">Event Managment &copy; 2017</p>
    </footer>
  </div>

);

export default footer;
