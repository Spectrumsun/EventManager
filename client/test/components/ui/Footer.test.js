import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../src/components/UI/Footer';

describe('<Footer />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('should render the footer', () => {
    shallow(<Footer />);
  });

  it('should render initial layout of footer', () => {
    expect(wrapper.getElements('ul')).toMatchSnapshot();
  });

  it('should render initial layout of footer', () => {
    expect(wrapper.getElements('div')).toMatchSnapshot();
  });

  it('should have one image on layout', () => {
    expect(wrapper.find('img').length).toEqual(1);
  });
});
