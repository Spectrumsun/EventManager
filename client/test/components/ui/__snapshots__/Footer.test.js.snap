// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render initial layout of footer 1`] = `
Array [
  <div
    className="center"
  >
    <footer>
      <div>
        <h3
          className="center"
        >
          Event Manager
        </h3>
      </div>
      <div
        style={
          Object {
            "display": "inline-block",
            "textAlign": "left",
          }
        }
      >
        <div>
          <div
            className="float-left"
          >
            <a
              className="navbar-brand"
              href="/"
            >
              <img
                alt=""
                height="30"
                src="https://res.cloudinary.com/skybound/image/upload/v1522444986/eventmanager/static/event.png"
                width="30"
              />
            </a>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                Social Media
              </p>
              <li
                className="footerColor"
              >
                facebook
              </li>
              <li
                className="footerColor"
              >
                twitter
              </li>
              <li
                className="footerColor"
              >
                instargram
              </li>
            </ul>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                Event manager
              </p>
              <li
                className="footerColor"
              >
                Questions
              </li>
              <li
                className="footerColor"
              >
                Jobs
              </li>
              <li
                className="footerColor"
              >
                Help
              </li>
            </ul>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                About
              </p>
              <li
                className="footerColor"
              >
                desgin by Taiwo Sunday
              </li>
              <li
                className="footerColor"
              >
                +2348038512455
              </li>
              <li
                className="footerColor"
              >
                https://github.com/Spectrumsun
              </li>
              <li
                className="footerColor"
              >
                taiwo.sunday@outlook.com
              </li>
            </ul>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                Site
              </p>
              <li
                className="footerColor"
              >
                Home
              </li>
              <li
                className="footerColor"
              >
                Centers
              </li>
              <li
                className="footerColor"
              >
                Events
              </li>
            </ul>
          </div>
        </div>
      </div>
      <p
        className="center"
      >
        Event Managment © 2017
      </p>
    </footer>
  </div>,
]
`;

exports[`<Footer /> should render initial layout of footer 2`] = `
Array [
  <div
    className="center"
  >
    <footer>
      <div>
        <h3
          className="center"
        >
          Event Manager
        </h3>
      </div>
      <div
        style={
          Object {
            "display": "inline-block",
            "textAlign": "left",
          }
        }
      >
        <div>
          <div
            className="float-left"
          >
            <a
              className="navbar-brand"
              href="/"
            >
              <img
                alt=""
                height="30"
                src="https://res.cloudinary.com/skybound/image/upload/v1522444986/eventmanager/static/event.png"
                width="30"
              />
            </a>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                Social Media
              </p>
              <li
                className="footerColor"
              >
                facebook
              </li>
              <li
                className="footerColor"
              >
                twitter
              </li>
              <li
                className="footerColor"
              >
                instargram
              </li>
            </ul>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                Event manager
              </p>
              <li
                className="footerColor"
              >
                Questions
              </li>
              <li
                className="footerColor"
              >
                Jobs
              </li>
              <li
                className="footerColor"
              >
                Help
              </li>
            </ul>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                About
              </p>
              <li
                className="footerColor"
              >
                desgin by Taiwo Sunday
              </li>
              <li
                className="footerColor"
              >
                +2348038512455
              </li>
              <li
                className="footerColor"
              >
                https://github.com/Spectrumsun
              </li>
              <li
                className="footerColor"
              >
                taiwo.sunday@outlook.com
              </li>
            </ul>
          </div>
          <div
            className="float-left"
          >
            <ul>
              <p
                className="footerTitle"
              >
                Site
              </p>
              <li
                className="footerColor"
              >
                Home
              </li>
              <li
                className="footerColor"
              >
                Centers
              </li>
              <li
                className="footerColor"
              >
                Events
              </li>
            </ul>
          </div>
        </div>
      </div>
      <p
        className="center"
      >
        Event Managment © 2017
      </p>
    </footer>
  </div>,
]
`;
